src/features/unified-share-modal/SharedLinkPermissionMenu.js

```diff
@@ -77,7 +80,24 @@ class SharedLinkPermissionMenu extends Component<Props> {
         };
 
         return (
-            <DropdownMenu constrainToWindow>
+            <DropdownMenu
+                constrainToWindow
+                onMenuClose={() => {
+                    if (canShowTag && sharedLinkEditTagTargetingApi) {
+                        sharedLinkEditTagTargetingApi.onComplete();
+                    }
+                }}
+                onMenuOpen={() => {
+                    if (canShowTag && sharedLinkEditTagTargetingApi) {
+                        sharedLinkEditTagTargetingApi.onShow();
+                    }
+
+                    // complete tooltip FTUX on opening of dropdown menu
+                    if (canShowTooltip && sharedLinkEditTooltipTargetingApi) {
```

Comment:
Can you pass the value of `isFtuxTooltipShown` from `SharedLinkSection` and also check for that here